src/main/java/org/testng/internal/PackageUtils.java

```diff
@@ -83,68 +84,92 @@ public static void addClassLoader(final ClassLoader loader) {
       }
     }
 
+    ParamContainer container = new ParamContainer(included,excluded,vResult);
```

Comment:
`params` looks to be a better name but I'm not sure a container is useful for only 3 params.